src/MQTTProtocolOut.c

```diff
@@ -211,18 +211,18 @@ int MQTTProtocol_setHTTPProxy(Clients* aClient, char* source, char** dest, char*
  */
 #if defined(OPENSSL)
 #if defined(__GNUC__) && defined(__linux__)
-int MQTTProtocol_connect(const char* ip_address, Clients* aClient, int ssl, int websocket, int MQTTVersion,
+int MQTTProtocol_connect(const char* ip_address, Clients* aClient, int unixsock, int ssl, int websocket, int MQTTVersion,
```

Comment:
`ip_address` is not just an IP address, it can be a path to a unix socket, please, can you rename it accordingly ?